shared/teams/team/rows/invite-row/request/index.tsx

```diff
@@ -176,7 +176,7 @@ const TeamRequestRowNew = (props: Props) => {
           {popup}
```

Comment:
IIRC surrounding the list item with a fragment and moving this outside works so the onclick handler doesn't need to care

Reply:
Is that a tip or a recommendation? That feels fragile if a refactorer doesn't know why there's a fragment.